services/common/src/components/projectSummary/AuthorizationSupportDocumentUpload.tsx

```diff
@@ -0,0 +1,129 @@
+import React, { FC } from "react";
+import { Field, getFormValues } from "redux-form";
+import { DOCUMENT, EXCEL, IMAGE, SPATIAL } from "@mds/common/constants/fileTypes";
+import { MINE_REPORT_DOCUMENT, PROJECT_SUMMARY_DOCUMENTS } from "@mds/common/constants/API";
+import RenderFileUpload from "../forms/RenderFileUpload";
+import { required, requiredNewFiles } from "@mds/common/redux/utils/Validate";
+import { IProjectSummaryDocument, IMineDocument } from "../..";
+import { useSelector } from "react-redux";
+import { FORM } from "../..";
```

Comment:
A bit nitpicky, but there are a few unused imports that could be removed here.

Reply:
> A bit nitpicky, but there are a few unused imports that could be removed here.

Updated accordingly.